sample-code/examples/node/helpers/apps.js

```diff
@@ -1,5 +1,5 @@
 if (process.env.DEV) {
-  exports.iosTestApp = "sample-code/apps/TestApp/build/release-iphonesimulator/TestApp.app";
+  exports.iosTestApp = "sample-code/apps/TestApp/build/release-iphonesimulator/TestApp-iphonesimulator.app";
```

Comment:
`ios-test-app`'s script generate `TestApp-iphonesimulator.app`. So, I change related lines to fit sample code.